app/services/staff_action_logger.rb

```diff
@@ -607,8 +599,33 @@ def log_web_hook(web_hook, action, opts = {})
     ))
   end
 
+  def log_embeddable_host(embeddable_host, action, opts = {})
+    old_values, new_values = get_changes(opts[:changes])
+  
+    UserHistory.create!(params(opts).merge(
+      action: action,
+      context: "host: #{embeddable_host.host}",
+      previous_value: old_values&.join(", "),
+      new_value: new_values&.join(", ")
+    ))
+  end
+
   private
 
+  def get_changes(changes)
+    return unless changes
+
+    changes.reject! { |k, v| k == "updated_at" }
+    old_values = []
+    new_values = []
+    changes.each do |k, v|
+      old_values << "#{k}: #{v[0]}"
+      new_values << "#{k}: #{v[1]}"
+    end
+
+    return old_values, new_values
```

Comment:
Returning an array is shorter 😉 

```suggestion
    [old_values, new_values]
```